src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  filter: Status,
+  setFilter: (status: Status) => void
```

Comment:
```suggestion
  setFilter(status: Status): void
```

it's a shorter way to write function type